src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewAddressReportedProcessor.java

```diff
@@ -66,6 +66,8 @@ public UUID process(OutcomeSuperSetDto outcome, UUID caseIdHolder, String type)
     root.put("eventDate", eventDateTime);
     root.put("addressType", type);
     root.put("region", regionLookup(outcome.getOfficerId()));
+    root.put("usualResidents",
+        outcome.getCeDetails().getUsualResidents() != null ? outcome.getCeDetails().getUsualResidents() : "0");
```

Comment:
shouldn't this be in the if statement above where you check if getCeDetails is null

Reply:
nice spot